{% extends 'layout.html' %}
{% block body %}
<h2>About antiSMASH</h2>
<p>
antiSMASH allows the rapid genome-wide identification, annotation and
analysis of secondary metabolite biosynthesis gene clusters in
bacterial and fungal genomes.  It integrates and cross-links with a
large number of in silico secondary metabolite analysis tools that have
been <a
  href="http://www.ncbi.nlm.nih.gov/pubmed/16221976,19297688,17506888,17400247,12691745,19360130,17913739,20462861,18950525,15980457,18978015"
  target="_blank"> published earlier</a>.
</p>

<p>
antiSMASH is powered by several open source tools: <a
  href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&PAGE_TYPE=BlastDocs&DOC_TYPE=Download"
  target="_blank">NCBI BLAST+</a>, <a href="http://hmmer.janelia.org/"
  target="_blank">HMMer 3</a>, <a
  href="http://www.drive5.com/muscle/">Muscle 3</a>, <a
  href="http://www.cbcb.umd.edu/software/glimmer/"
  target="_blank">Glimmer 3</a>, <a
  href="http://www.microbesonline.org/fasttree/"
  target="_blank">FastTree</a>, <a
  href="http://treegraph.bioinfweb.info/" target="_blank">TreeGraph
  2</a>, <a href="http://ggasoftware.com/opensource/indigo/"
  target="_blank">Indigo-depict</a>, <a href="http://codeboje.de/pysvg/"
  target="_blank">PySVG</a> and <a href="http://keith-wood.name/svg.html"
  target="_blank">JQuery SVG</a>.
</p>

<p>
antiSMASH is the product of a collaborative effort between the <a
  href=" http://www.rug.nl/gbb/micphys" target="_blank">Department of
  Microbial Physiology</a> and <a href=" http://www.rug.nl/gbic"
  target="_blank">Groningen Bioinformatics Centre</a> of the University
of Groningen, the <a href="
  http://www.imit.uni-tuebingen.de/abteilungen/mikrobiologiebiotechnologie/work-groups/weber.html"
  target="_blank">Department of Microbiology</a> of the University of
T&uuml;bingen, the <a href="http://www.mib.ac.uk/people/profile/?alias=takanoe"
  target="_blank"> Synthetic Biology </a> and <a href="http://www.ls.manchester.ac.uk/research/researchgroups/computationalandevolutionarybiology/people/?alias=breitlingr"
  target="_blank"> Computational and Evolutionary Biology </a>
  groups at the Faculty of Life Sciences at the University of Manchester
and the <a href=" http://www.fischbachgroup.org/"
  target="_blank">Department of Bioengineering and Therapeutic
  Sciences</a> at the University of California, San Francisco.
</p>

<p>
antiSMASH is supported by the GenBiotics program of the Dutch
Technology Foundation (STW), which is the applied-science division of
The Netherlands Organisation for Scientific Research (NWO) and the
Technology Programme of the Ministry of Economic Affairs (grant STW
10463) as well as the GenBioCom program of the German Ministry of
Education and Research (BMBF) grant 0315585A.
</p>

<p>
Publications:<br>
Kai Blin, Marnix H. Medema, Daniyal Kazempour, Michael A. Fischbach,
Rainer Breitling, Eriko Takano, &amp; Tilmann Weber (2013).
antiSMASH 2.0 â€” a versatile platform for genome mining of secondary
metabolite producers.<em>Nucleic Acids Research</em>
<a href="http://dx.doi.org/10.1093/nar/gkt449">doi: 10.1093/nar/gkt449</a>.<br>
<br>
Marnix H. Medema, Kai Blin, Peter Cimermancic, Victor de Jager, Piotr
Zakrzewski, Michael A. Fischbach, Tilmann Weber, Rainer Breitling &amp;
Eriko Takano (2011). antiSMASH: Rapid identification, annotation and
analysis of secondary metabolite biosynthesis gene clusters. <em>Nucleic
  Acids Research</em> 39: W339-W346.
</p>
{% endblock %}
